### ISSUE TITLE
No i18n support

### ISSUE THREAD

**Issue by yarons** [AlmaLinux/wiki#63]
There's no way to easily translate this wiki to other languages, maybe we should consider a different solution for that because an international community requires different working patterns to develop in parallel.

**Comment by srbala** [comment]
@yarons Site is developed using `vuepress 1.x` does have `i18n` support. https://v1.vuepress.vuejs.org/guide/i18n.html#site-level-i18n-config 

Would like to check it would work. Base project settings might need some enhancement or adding additional packages.

**Comment by yarons** [comment]
OK, so now I need to check if there's an easy solution to use a localization platform to make this process as fluent as possible.
I'll be looking at Weblate mostly but I think Crowdin was already selected.

**Comment by yarons** [comment]
Basically we can start setting this up, I think Crowdin supports that natively but it's not open source, I can check with Weblate if it's also possible to work with their platform (It's less convenient for documents).

Meanwhile I've the following bug:
https://github.com/vuejs/vuepress/issues/2944

**Comment by bennyvasquez** [comment]
I just realized that #420 is a duplicate of this one, but the other one has more recent activity on it. @yarons would you mind if I close this one in favor of that one? There's an active pull request related to it that attempts to fix the problem.

**Comment by yarons** [comment]
As you wish.

**Comment by bennyvasquez** [comment]
Sounds good! Sorry for missing the duplicate before, and please do follow the new one as well. We did end up choosing weblate, so we'd welcome any expertise you have!
